kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/food2door/Supplier.java

```diff
@@ -0,0 +1,9 @@
+package com.kodilla.good.patterns.food2door;
+
+import java.util.List;
+import java.util.Map;
+
+interface Supplier {
+List productList();
```

Comment:
this should be list of products, not just List. You should always specify generic type and write in this case: List<Product>